packages/react-core/src/components/Pagination/examples/PaginationTop.tsx

```diff
@@ -26,6 +26,7 @@ export const PaginationTop: React.FunctionComponent = () => {
       onSetPage={onSetPage}
       widgetId="top-example"
       onPerPageSelect={onPerPageSelect}
+      ouidId="PaginationTop"
```

Comment:
The build is failing because this is a typo :)
it is `ouidId` but should be `ouiaId`

Reply:
@nicolethoen thank you! fixed!